shopedia/src/auth/logout.php

```diff
@@ -1,5 +1,6 @@
 <?php
     // Clearing the session variables
+    session_start();
```

Comment:
I'm using the newer version of php for the Dockerized version which requires the session to be initialized first before it can be destroyed.